packages/react-portal/src/components/Portal/renderPortal.tsx

```diff
@@ -1,10 +1,11 @@
 import * as ReactDOM from 'react-dom';
+import * as React from 'react';
 import { PortalState } from './Portal.types';
 
 /**
  * Render the final JSX of Portal
  */
-export const renderPortal = (state: PortalState) => {
+export const renderPortal: React.FC<PortalState> = (state: PortalState) => {
```

Comment:
This is a render function, now a react component
```suggestion
export const renderPortal = (state: PortalState) => {
```

Reply:
Was fixed in a different comment.